src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1500,18 +1505,18 @@ public Iterable<AgentRole> getCustomAgentRoles() {
 
   public Iterable<org.zendesk.client.v2.model.Request> getRequests() {
     return new PagedIterable<>(
-        cnst("/requests.json"), handleList(org.zendesk.client.v2.model.Request.class, "requests"));
+        cbp("/requests.json"), handleList(org.zendesk.client.v2.model.Request.class, "requests"));
```

Comment:
Not in the list provided in [support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z](https://support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z)

The endpoint documentation doesn't include CBP either https://developer.zendesk.com/api-reference/ticketing/tickets/ticket-requests/#list-requests